kubejs/startup_scripts/item_registry.js

```diff
@@ -351,99 +365,99 @@ onEvent('item.registry', (event) => {
+        { name: 'frosty', band: 0xb9d3ec, outline: 0xb9d3ec },
+        { name: 'gold', band: 0xffdc00, outline: 0xffdc00 },
+        { name: 'iron', band: 0xffcc99, outline: 0xffcc99 },
+        { name: 'lead', band: 0x333952, outline: 0x333952 },
+        { name: 'nickel', band: 0x8c8f86, outline: 0x8c8f86 },
+        { name: 'osmium', band: 0xc1e0e7, outline: 0xc1e0e7 },
+        { name: 'regal', band: 0xebbe76, outline: 0xebbe76 },
+        { name: 'silver', band: 0xc5d1fe, outline: 0xc5d1fe },
+        { name: 'tin', band: 0x7d81a4, outline: 0x7d81a4 },
+        { name: 'uranium', band: 0x333c2f, outline: 0x333c2f },
+        { name: 'zinc', band: 0x818653, outline: 0x818653 },
         // natural
-        {name: 'brutish_zombee', band: 0x2f4e32, outline: 0x2f4e32},
-        {name: 'clogged',        band: 0xba9d7b, outline: 0xba9d7b},
-        {name: 'coal',           band: 0x303030, outline: 0x303030},
-        {name: 'creeper',        band: 0x0c9f0a, outline: 0x0c9f0a},
-        {name: 'ender',          band: 0x339786, outline: 0x339786},
-        {name: 'forest',         band: 0x4c4c26, outline: 0x4c4c26},
-        {name: 'glowstone',      band: 0xffbc5e, outline: 0xffbc5e},
-        {name: 'icy',            band: 0x92b9fe, outline: 0x92b9fe},
-        {name: 'obsidian',       band: 0x663399, outline: 0x663399},
-        {name: 'pigman',         band: 0x885956, outline: 0x885956},
-        {name: 'rgbee',          band: 0xffffff, outline: 0xffffff},
-        {name: 'rocky',          band: 0x8f8f8f, outline: 0x8f8f8f},
-        {name: 'sand',           band: 0xe7e4bb, outline: 0xe7e4bb},
-        {name: 'skeleton',       band: 0xf6f2e6, outline: 0xf6f2e6},
-        {name: 'slimy',          band: 0x73c262, outline: 0x73c262},
-        {name: 'water',          band: 0x5979ef, outline: 0x5979ef},
-        {name: 'zombie',         band: 0x2f4e32, outline: 0x2f4e32},
+        { name: 'brutish_zombee', band: 0x2f4e32, outline: 0x2f4e32 },
+        { name: 'clogged', band: 0xba9d7b, outline: 0xba9d7b },
+        { name: 'coal', band: 0x303030, outline: 0x303030 },
+        { name: 'creeper', band: 0x0c9f0a, outline: 0x0c9f0a },
+        { name: 'ender', band: 0x339786, outline: 0x339786 },
+        { name: 'forest', band: 0x4c4c26, outline: 0x4c4c26 },
+        { name: 'glowstone', band: 0xffbc5e, outline: 0xffbc5e },
+        { name: 'icy', band: 0x92b9fe, outline: 0x92b9fe },
+        { name: 'obsidian', band: 0x663399, outline: 0x663399 },
+        { name: 'pigman', band: 0x885956, outline: 0x885956 },
+        { name: 'rgbee', band: 0xffffff, outline: 0xffffff },
+        { name: 'rocky', band: 0x8f8f8f, outline: 0x8f8f8f },
+        { name: 'sand', band: 0xe7e4bb, outline: 0xe7e4bb },
+        { name: 'skeleton', band: 0xf6f2e6, outline: 0xf6f2e6 },
+        { name: 'slimy', band: 0x73c262, outline: 0x73c262 },
+        { name: 'water', band: 0x5979ef, outline: 0x5979ef },
+        { name: 'zombie', band: 0x2f4e32, outline: 0x2f4e32 },
```

Comment:
monospace murderer 👀

Reply:
Blame Prettier :P

Reply:
More like _thank_ Prettier

Reply:
I like the monospace, but I like identical formatting everywhere more :D

Reply:
felt neat to have the hex codes of the bees line up tho, but excluding just this block would require a comment 😅